### PR TITLE
bfs

### PR DESCRIPTION
**Pull request by stacyburris**
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

File: javascript/code-challenges/breadthFirst/breadth-first.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      } else {
+        visitedNodes.add(neighborNode);
+      }
+      queue.push(neighborNode);
+    }
+  }
+  return visitedNodes;
+}
+
+module.exports = { breadthFirstSearch, Graph };
```
Comment (adrianhuebner) [inline javascript/code-challenges/breadthFirst/breadth-first.js:27]: Code is looking good, take your time with figuring out tests!
